server/models/alumniSchema.js

```diff
@@ -14,7 +29,7 @@ const alumniSchema = new Schema(
     college: {type: String, required: false},
     //requests: [{type: Schema.Types.ObjectId, ref: 'requestSchema'}]
     //posts: [{type: Schema.Types.ObjectId, ref: 'postSchema'}]
-    availabilities: [Date],
+    availabilities: [timeAvailabilitySchema],
```

Comment:
`availabilites: [timeAvailabilitySchema]` probably won't work they way you want it to work.

If you want it to be a schema, it is easiest to store it as `availabilities: [{type: Schema.Types.ObjectId, ref: 'timeAvailabilitySchema'}]` - but I'm still not sure it makes sense to store availabilities elsewhere

As an object, it could just be `availabilities: [{type: Object, required: false}]

Reply:
This uses subdocuments, a newer feature not used as frequently in tutorials